src/libical/icalparameter.h

```diff
@@ -571,7 +571,7 @@ LIBICAL_ICAL_EXPORT int icalparameter_has_same_name(icalparameter *param1, icalp
 /**
  * @brief Returns a string representing the given ::icalparameter_kind
  * @param kind The icalparameter_kind
- * @return A string representing kind
+ * @return A string representing @a kind
```

Comment:
why @ a ?

Reply:
Good question. I don't remember doing that... Seems like it should be a `@p` reference. I guess this raises another thing I could do, to change places where `@a` refers to a parameter to `@p`.

Reply:
On second thoughts, I'm leaving this as `@a`. That seems to be how the rest of the docs reference parameters, so this seems like a good addition. If that's good with you.

Reply:
yep